axelor-account/src/main/resources/reports/Invoice.rptdesign

```diff
@@ -4775,10 +4775,34 @@ row["partnerFirstName"]
                                 <cell id="4459">
                                     <property name="colSpan">1</property>
                                     <property name="rowSpan">1</property>
-                                    <label id="4420">
+                                    <text id="5370">
                                         <property name="textAlign">left</property>
-                                        <text-property name="text">Acompte(s) lié(s)</text-property>
-                                    </label>
+                                        <property name="dataSet">TranslateDS</property>
+                                        <list-property name="paramBindings">
+                                            <structure>
+                                                <property name="paramName">param_1</property>
+                                                <simple-property-list name="expression">
+                                                    <value type="javascript">'invoice.advancePayments'</value>
```

Comment:
Wrong translation key, it should be 'Invoice.advancePayments' instead of 'invoice.advancePayments'